src/components/Tooltip/Tooltip.module.css

```diff
@@ -1,72 +1,126 @@
-@import '../../design-tokens/mixins.css';
+/* stylelint-disable selector-pseudo-class-no-unknown */
 
 /*------------------------------------*\
     #TOOLTIP
 \*------------------------------------*/
 
 /**
- * 1) An icon that appears 
+ * 1) Tippyjs provides .tippy-box, which has two child elements: .tippy-content and .tippy-arrow
+ * 2) Tippyjs also attaches data-attribute `data-placement` depending on how the tooltip is aligned
  */
+
 .tooltip {
-  display: inline-block;
-  position: relative;
-  overflow: visible;
+  border-style: solid;
+  border-width: var(--eds-border-width-sm);
+  border-radius: var(--eds-border-radius-lg);
+  box-shadow: var(--eds-theme-box-shadow);
```

Comment:
I think this shadow is thinner than the original shadow, but using what is available for consistency
original on the left, this pr on the right:
![image](https://user-images.githubusercontent.com/86632227/157144450-6d413e75-161d-4af8-b42a-5c5af7dce0f5.png)

Reply:
I wonder if we need a ticket for updating the tokens 🤔

Reply:
I think all tokens are subject to change while design is doing discovery right now; so we can feel free to replace the values with our existing ones

Reply:
Sorry, to clarify, are you saying to add a token to `shadows.json`, or not using a token for now and assigning a value to `box-shadow`?

Reply:
either add a token to `shadows.json` or update `eds-theme-box-shadow` to match ours 🙂 we should always be using a token, but the values are owned by us, and will probably evolve over the next few weeks. so not sure it's worth to update all of them up front, but if we notice discrepancies we can default to the values we were already using for the component

Reply:
Ah thanks for the explanation, will update shadow token `md`,

Reply:
@anniehu4 @jinlee93 @dierat yep the token work is firming up, and I'm going to take the lead on an initial import of what they've put in place in Figma. Hoping to get that in there by the end of the day.